### DISCUSSION TITLE
OOPS screen while trying to edit the connections on Airflow UI

### DISCUSSION THREAD

**Discussion by abheast11** [apache/airflow#31607] [Q&A]
<img width="1440" alt="Screenshot 2023-05-30 at 12 28 23 PM" src="https://github.com/apache/airflow/assets/135002000/4871f3fa-77e0-4495-9d15-f41c61037613">

<img width="1440" alt="Screenshot 2023-05-30 at 12 25 31 PM" src="https://github.com/apache/airflow/assets/135002000/522a552a-1142-4eb2-bd1e-f2f4ef394d77">

Is this related any installation of airflow packages? If yes, how do I proceed further with this issue.

Thanks for your help.

**Comment by potiuk** [comment]
You need to look at webserver logs and see detailed error: The error message is clear there if you read it:

* gather the relevant information (detailed logs with error, ....

We cannot print detailed error here, because of security concerns (there are certain attacks that reveal too much information). So look at the logs of the webserver, find the details about the issue and post it here if you need help.

**Reply by abheast11** [reply]
ModuleNotFoundError: No module named 'airflow.providers.apache'

Received the following error while traversing through the logs of the webserver.

**Reply by abheast11** [reply]
Hi potiuk,

Managed to fix the error. Thank you for helping me out.
